README.md

```diff
@@ -297,6 +297,27 @@ of mangled property names.
 Using the name cache is not necessary if you compress all your files in a
 single call to UglifyJS.
 
+When using `minify()`, you may specify `--name-cache` in the following way:
```

Comment:
Done in https://github.com/mishoo/UglifyJS2/pull/2176/commits/ec66ec295f9e32e5ef3028a6d919ae3ac61a3d9c

I removed the mention of `minify()` since it feels superfluous under the section we've migrated over.